lib/Partitioner/Partitioner.cpp

```diff
@@ -914,7 +914,40 @@ Partitioner::backendBasedPartition(Function *F,
   for (auto &N : F->getNodes()) {
     for (auto &backend : backends) {
       // Find the first backend that supports this node. The order of backends
-      // is important.
+      // is important. The check flow is :
+
+      // Step 1: If a node is in pre-defined non-supported nodes set, it can not
```

Comment:
Maybe it's just my preference but here I would find it easier to read the "step N" comments inline with the code, rather than as a separate block that I need to scroll up to reference.  And that way we could get rid of the "step N" tags themselves, which have to be kept up to date over time.

Reply:
Done!